### DISCUSSION TITLE
Awesome Foundry: a repo with a list of tools, libraries and templates to use with Foundry.

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#303] [General]
Recently I found this repo: https://github.com/crisgarner/awesome-foundry, that has a lot of material targeting Foundry, the repo also contains a lot of tutorials e.g. invariant testing, to reproduce hacks, etc...

Another cool fact is that most of the tools are not old and are being actively maintained.

If you know about similar repos, please share in the comments.

**Comment by pyromaus** [comment]
thats sick, thanks bro

**Comment by cromewar** [comment]
Pinning this for a while, thanks @pacelliv

**Comment by john-lawniczak** [comment]
Whoah daddy! 

Just discovering this, incredible! Any tools in particular you like on here so far @pacelliv?

**Reply by ghost** [reply]
I've only focused on the tutorials for now, some are very good for example the invariant test for the WETH token.
